applications/Chat/evaluate/utils.py

```diff
@@ -72,17 +72,6 @@ def get_data_per_category(data, categories):
     return data_per_category
 
 
-def remove_articles(text: str) -> str:
-    """
-    Remove articles "a, an, the" in the given text.
-    It is used in evaluation of automatic metrics.
-
-    """
-
-    pattern = re.compile(r"\b(a|an|the)\b", re.UNICODE)
-    return re.sub(pattern, " ", text)
-
-
```

Comment:
Why do we remove this part?

Reply:
Because it will delete the model answers with only letter a. For example, the instruction is a multiple choice questions, so the model answer may be "a", which will be removed as articles